rpqueue/__init__.py

```diff
@@ -932,6 +932,13 @@ def execute_task_threads(queues=None, threads=1, wait_per_thread=1, module=None)
     signal.signal(signal.SIGUSR2, _print_stackframes_on_signal)
     if module:
         __import__(module)
+
+        try:
+            # Call the function to setup a LD client specificly for tasks
+            # https://chownow.atlassian.net/browse/CN-25221
+            __import__(module).set_ld_task_client()
+        except Exception as e:
+            log_handler.exception("ERROR: Unable to setup the LD Task Client: %s", e)
```

Comment:
Main question is, what do we want to occur if this fails? I'm assuming we _don't_ want to just exit everything wholesale, but if we move forward, there's the possibility that LD flags may be changed during the lifespan of a process and would result in misbehaving tasks. But maybe that's a fair trade off?

Reply:
Yeah I think monitoring for this would make sense.. under the hood the LD SDK attempts to initialize with the "live" config from LD's servers, but if it fails it will fallback to using the cached values. If THAT fails, it will use the fallback values that our code uses for each flag. This behavior is similar to the other parts of the app so shouldn't be an issue.. I suggest just updating the exception message to something like `ERROR: Unable to setup the LaunchDarkly SDK within Rpqueue:` so that we know where it's coming from